flank-scripts/src/main/kotlin/flank/scripts/shell/firebase/sdk/CheckForSDKUpdateCommand.kt

```diff
@@ -0,0 +1,71 @@
+package flank.scripts.shell.firebase.sdk
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.kittinunf.fuel.Fuel
+import flank.common.downloadFile
+import flank.scripts.shell.utils.currentPath
+import flank.scripts.utils.parseToVersion
+import kotlinx.coroutines.runBlocking
+import java.nio.file.Paths
+import java.time.Instant
+
+private const val RAW_GITHUB = "https://raw.githubusercontent.com/Flank/gcloud_cli"
+
+object CheckForSDKUpdateCommand : CliktCommand(
+    name = "checkForSdkUpdate",
+    help = "Verifies if there were changes in gcloud sdk that need to be implemented in flank"
+) {
+
+    private val githubToken by option(help = "Git Token").required()
+    private val zenhubToken by option(help = "Zenhub Token").required()
+    private val lastRun by lazy { runBlocking { getLastSDKUpdateRunDate(githubToken) } }
+    private val openedUpdates by lazy { runBlocking { checkForOpenedUpdates(githubToken) } }
+
+    override fun run() = runBlocking {
+        println("** Find previously checked commit")
+        val oldSha = getCommitsUntilLastCheck(githubToken, lastRun) ?: run {
+            println("** Unable to find previous commit")
+            return@runBlocking
+        }
+        println("** Find latest commit")
+        val newSha = getCommitsUntilLastCheck(githubToken, Instant.now().toString()) ?: run {
+            println("** Unable to find latest commit")
+            return@runBlocking
+        }
+
+        if (oldSha == newSha) {
+            println("** No new commits since the last run")
+            return@runBlocking
+        }
+
+        with(createContext(oldSha)) {
+            println("** New version $newVersion")
+            println("** Old version $oldVersion")
+            when {
+                oldVersion < newVersion && openedUpdates != null -> updateOpenedEpic()
+                oldVersion < newVersion && openedUpdates == null -> createEpicIssue()
+                else -> run {
+                    println("** No new features")
+                    return@runBlocking
+                }
+            }
```

Comment:
We could move `oldVersion < newVersion` to top to make code smaller

```kotlin
when {
    oldVersion >= newVersion -> run {
                    println("** No new features")
                    return@runBlocking
                }
    openedUpdates != null -> updateOpenedEpic()
    openedUpdates == null -> createEpicIssue()
            }
`

Reply:
Good idea, forgot `when` is not an expression in that case and don't have to be exhaustive 😅